leakcanary-android-core/src/main/java/leakcanary/internal/AndroidHeapDumper.kt

```diff
@@ -84,15 +84,15 @@ internal class AndroidHeapDumper(
 
     val toast = waitingForToast.get()
 
-    try {
-      Debug.dumpHprofData(heapDumpFile!!.absolutePath)
+    return try {
```

Comment:
IJ keeps suggesting that and honestly that one looks hard to read to me

Reply:
Same but I'm getting used to it.